src/endpoints/accounts/entities/account.esdt.history.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class AccountEsdtHistory {
```

Comment:
I think this class should be inherited from AccountHistory and just add the token property in addition.